packages/contracts-bedrock/test/libraries/Predeploys.t.sol

```diff
@@ -150,12 +157,28 @@ contract Predeploys_UnclassifiedInterop_Test is Predeploys_TestInit {
     /// @notice Tests that the predeploy addresses are set correctly. They have code and the
     ///         proxied accounts have the correct admin. Using interop with inbox.
     function test_predeploysWithInbox_succeeds() external {
-        _test_predeploys(Fork.INTEROP, true);
+        _test_predeploys(Fork.INTEROP, true, false);
     }
 
     /// @notice Tests that the predeploy addresses are set correctly. They have code and the
     ///         proxied accounts have the correct admin. Using interop without inbox.
     function test_predeploysWithoutInbox_succeeds() external {
-        _test_predeploys(Fork.INTEROP, false);
+        _test_predeploys(Fork.INTEROP, false, false);
+    }
+}
+
+/// @title Predeploys_CustomGasToken_Test
+/// @notice Tests the `Predeploys` contract with custom gas token.
+contract Predeploys_CustomGasToken_Test is Predeploys_TestInit {
+    /// @notice Test setup. Enabling custom gas token.
+    function setUp() public virtual override {
+        super.enableCustomGasToken();
+        super.setUp();
+    }
+
+    /// @notice Tests that the predeploy addresses are set correctly. They have code and the
+    ///         proxied accounts have the correct admin. Using custom gas token.
+    function test_predeploysWithCustomGasToken_succeeds() external {
```

Comment:
Nit - can be included in the original test contract with `skipIfDevFeatureDisabled`